libraries/core/commands/setCookie.js

```diff
@@ -2,19 +2,20 @@ import AppCommand from '../classes/AppCommand';
 
 /**
  * Sends an setCookie command to the app.
- * @param {Object} params The command parameters.
- * @param {string} params.name The name of the cookie.
- * @param {string} params.value The value of the cookie.
- * @param {string} params.domain The domain for the cookie.
- * @param {number} params.expires The expiration datetime.
- * @param {string} params.path The path for the cookie.
- * @param {boolean} params.secure
- * @param {boolean} params.httpOnly
+ * @param {Object} params The command parameters
+ * @param {string} params.name The name of the cookie
+ * @param {string} params.value The value of the cookie
+ * @param {string} params.domain The domain for the cookie
+ * @param {int} params.expires The expires datetime
+ * @param {string} params.path The path for the cookie
+ * @param {Boolean} params.secure Set is secure
```

Comment:
it needs to be `boolean`. ESLint also complains here ;) 
Please make sure to run eslint always before you commit something to avoid this